src/components/LongitudinalSection.js

```diff
@@ -121,22 +107,9 @@ function Longitudinal({ className, selectedSection, coverageData, data }) {
         yKey="coverage"
         hexColor={sectionLineColors[selectedSection]}
       />
-      <div className="flex items-center justify-center">
-        {/* hiding for now */}
-        <Metrics
-          percentage={percentage}
-          rotation="180deg"
-          color="#d24200"
-          fraction={sectionFractions[selectedSection]}
-          title="Current"
-        />
-        <Metrics
-          percentage={percentage}
-          rotation="0deg"
-          color="#26c485"
-          fraction={sectionFractions[selectedSection]}
-          title="Monthly Average"
-        />
+      <div className="flex items-start justify-center">
+        <Metrics percentage={percentage} up={isGreater} fraction={sectionFractions[selectedSection]} title="Current" />
```

Comment:
There seems to be a small typo here causing the icon to only display as a down arrow. `up=` should be changed to `trendUp=`

Reply:
Haha, oops, this is the danger of changing a variable name mid way through a task